src/docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -5,23 +5,28 @@ redirect_from:
   - /data-management/rollups/
   - /learn/rollups/
   - /data-management/event-grouping/
-description: "Learn about fingerprints, the grouping algorithm, grouping by stack trace, grouping by exceptions, and fallback grouping."
+  - /guides/grouping-and-fingerprints/
+  - /product/sentry-basics/guides/grouping-and-fingerprints/
+description: "Learn about fingerprints, the grouping algorithm, grouping by stack trace, grouping by exceptions, and fallback grouping. Learn more about how Sentry groups issues together as well as different approaches for updating how events group into issues"
 ---
 
-All events have a fingerprint. Events with the same fingerprint are grouped together into an issue.
+A fingerprint is a way to uniquely identify an event, and all events have one. Events with the same fingerprint are grouped together into an issue. So if two events have the same stack trace, then they will group into one issue.
```

Comment:
As multiple events can have the same fingerprint, I would not write "uniquely identify an event" here. Maybe "uniquely identify an error"?

Reply:
Also maybe we can remove the second sentence, because the same information is repeated in the next paragraph?